### DISCUSSION TITLE
Queue to receive messages not picked up by other queues

### DISCUSSION THREAD

**Discussion by ScottGibbons00** [rabbitmq/rabbitmq-server#4232] [Questions] [answered]
Hello,

I am new to learning RabbitMQ, so please excuse me if this question is common. Am I able to configure a queue for all messages that do not go to any other queue? For example, I have one queue with binding key `apples.*` and another queue with binding key `bananas.*` and I am using topic exchange. If I send a message `pears.eat` or `mango.slice` to RabbitMQ, I would want these to go to this third queue. But messages like `apples.eat` would go to the first queue and not the third queue.

Thanks in advance!

**Comment by mkuratczyk** [comment, marked as answer]
Hi,

Yes, you can configure an alternate exchange: https://www.rabbitmq.com/ae.html. Once you configure it for the exchange you already have and bind the third queue to this new exchange, messages that don't match any of the bindings (neither `apples.*` nor `bananas.*`) will be routed through the alternate exchange to the third queue.

**Reply by ScottGibbons00** [reply]
Perfect! Thank you for the help and quick response.
